docs/Authentication.md

```diff
@@ -8,6 +8,8 @@ There are three ways to authenticate with the Flowdock API:
 
 Most applications should use OAuth 2.0 as it doesn't store passwords locally and does not require users to have a separate password for Flowdock.
 
+Integrations can use [flow tokens](#source-token) from configured [sources](sources) to POST messages to flows and perform actions on the source.
```

Comment:
The same definition is also duplicated on this same page in the source token authentication section. The documentation will likely get out of date if the list of allowed actions changes. :grimacing: